addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -213,11 +213,22 @@ const wSnippetMenu = weSnippetEditor.SnippetsMenu.extend({
         if (HighlightOptionEl) {
             HighlightOptionEl.dataset.textSelector = HighlightOptionEl.dataset.selector;
         }
-        
+
         // TODO remove in master: see snippets.xml
         $html.find('we-checkbox[data-dependencies="!footer_copyright_opt"]')[0]?.remove();
         $html.find('[data-name="header_language_selector_none_opt"]')[0]?.remove();
         $html.find('we-select[data-dependencies="!header_language_selector_none_opt"]')[0]?.removeAttribute("data-dependencies");
+
+        // Replace the "src" attribute of iframes with "data-src" to prevent the
+        // iframes from loading in the snippet menu. This is a fix in stable
+        // that is no longer needed in later versions, where the snippets' HTML
+        // code is no longer rendered in the snippet menu.
+        const iframeEls = $html.find("[data-snippet] iframe[src]");
+        for (const iframeEl of iframeEls) {
```

Comment:
In fact: why are you implementing the system that change data-src to src in web_editor, but only using it in website? :thinking: 

Let's put everything in web_editor.

Reply:
> In fact: why are you implementing the system that change data-src to src in web_editor, but only using it in website?

Because no iframes are used in `mass mailling` snippets but indeed it's better to move it to `web_editor` 👍🏼